en/components/themes/theming-demo.md

```diff
@@ -0,0 +1,347 @@
+</div>
+```
+<div class="divider--half"></div>
+
+After that, in a new SCSS file set the wrapper element `class` to include the components with their themes.
+
+```scss
+// styles/dark-themes-class.scss
+@import 'dark-theme';
+
+:host {
+    display: block;
+    margin: 16px;
+    box-shadow: igx-elevation($elevations, 12);
+
+    .dark-theme {
+      background: $dark-color;
+        ::ng-deep {   
+          @include igx-grid($dark-grid-theme);
+          @include igx-snackbar($dark-snackbar-theme);
+          @include igx-input-group($dark-input-group-theme);
+          @include igx-grid-paginator($dark-grid-paginator-theme);
+          @include igx-button($dark-button-theme);
+          @include igx-dialog($dark-dialog-theme);
+          @include igx-grid-filtering($dark-grid-filtering-theme);
+          @include igx-button-group($dark-button-group-theme); 
+      }
+    }
+}
+```
+And the result is:
+
+<div class="divider--half"></div>
+<div class="sample-container" style="height: 650px">
+    <iframe id="dark-theme-sample-iframe" seamless width="100%" height="100%" frameborder="0" src="{environment:demosBaseUrl}/dark-theme-sample" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dark-theme-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}"> view on stackblitz
+    </button>
+</div>
+<div class="divider--half"></div>
+
+Import the *utilities*, component mixins and the theme functions, define the colors, define the themes and apply them. These are the steps for styling your app with **Ignite UI for Angular Theming**. 
+
+### Theme Chooser
+In the above sample we set only one theme per component.
+
+If we want to define more themes for a single **Ignite UI for Angular Control**, with the idea of changing them, we can add a **theme chooser**.
+
+This can be achieved with few modifications:
+- Create a SCSS file and define the classes, which will include the themes:
+
+```scss
+// // styles/themes-classes.scss
+@import 'themes';
+
+:host {
+    display: block;
+    margin: 16px;
+    box-shadow: igx-elevation($elevations, 12); 
+
+    // Set the light themes for the components. 
+    .light-theme {
+      background: $light-color;
+      ::ng-deep {   
+        @include igx-grid($light-grid);
+        @include igx-snackbar($light-snackbar); 
+        @include igx-input-group($light-input-group);
+        @include igx-grid-paginator($light-grid-paginator);
+        @include igx-button($light-button);
+        @include igx-dialog($light-dialog);
+        @include igx-grid-filtering($light-grid-filtering);
+        @include igx-button-group($light-button-group);     
+      }
+    }
+    ...
+  }
+```
+
+<div class="divider--half"></div>
+
+- Bind the wrapper class selector and set an event, which triggers the function that manipulates the `class` reference:
+
+```html
+<div [class] = "wrapper">
```

Comment:
Wouldn't it be better if we used HostBinding to bind to the class attribute inside the typescript class definition of the component for better readability?